tests/mocha/unit-tests/components/CoreFonts/core.font.container.spec.js

```diff
@@ -19,7 +19,7 @@ describe('<CoreFontContainer />', () => {
     consoleList = {}
     retry = []
 
-    mock.post('/saveCoreFont', (req) => {
```

Comment:
The removal of `req` appears unrelated to this PR. If that's the case, the change should be made in a different PR.

Reply:
Its required after running the phpstorm format code. And as I checked it is also no used elsewhere.

Reply:
Yes, but it's not relevant to this PR. As a rule, each PR / commit should have one purpose. In this case it's fixing up the hash history error message and anything unrelated to that should not be included.

Reply:
ok